games-rpg/runescape_launcher/runescape_launcher-0.9.0.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A launcher script for old-school Runescape"
+DESCRIPTION="This is a launcher script that downloads the \
+old school Runescape launcher and starts it. \
+Please note that runescape itself is closed-source and has a \
+proprietary license.\
+\
+This script is based on https://gitlab.com/coringao/runescape/"
+HOMEPAGE="https://github.com/TheZeldakatze/haiku-oldschoolrunescape-launcher"
+COPYRIGHT="2023, Maite Gamper \
+2016-2020, Carlos Donizete Froes"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/TheZeldakatze/haiku-oldschoolrunescape-launcher/archive/refs/tags/v0.9.0.tar.gz"
```

Comment:
Looking at your changes there I  don't really see the need to not use upstream 0.8 tag instead?
The commit you made and the rdef file could be easaly adopted on the original source I guess (haven't tested).

Reply:
I don't know if that would make sense, as the original script has quite a few differences compared to the stuff I did (though I'd guess it would be possible to do all the changes based directly on coringao/runescape, though you'd need to do this overflow buffer check thing that closes the game, as otherwise you'll have an unresponive window and you'll have to force-close the game).